worlds/sims4/Items.py

```
@@ -24,21 +25,25 @@ class Sims4Item(Item):
 
 skills_table: dict[int, ItemDict] = {}
 
+
 def add_skill(skill_name: str, expansion: str, max_level: int) -> int:
+    if max_level < 2:
+        raise ValueError(f"max_level must be at least 2, got {max_level}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)